src/sui-card/_sui-card.scss

```diff
@@ -1,46 +1,34 @@
 .sui-Card {
-  background-color: $c-bg-light;
-  border: 0;
-  display: flex;
-  flex-direction: column;
+  background-color: $c-gray-lighter;
 
+  &-primary {
+    margin: 0;
```

Comment:
Regarding margins and paddings remain exclusive for sui-Card implentations, not for the component as such, I suggest to make them themable by creating Sass variables (as it has been done for the background color).